README.md

```diff
@@ -1,29 +1,55 @@
-# Compliance Workshop Setup
+tf-compliance-workshop is a provisioner for AWS that uses Terraform for compliance workshops. It spins up one Automate server and the requisite number of workstations using the playing card system. It assumes you have terraform installed, your AWS ssh keys and secret keys, AWS CLI setup, etc. 
 
-## Things you'll need
+# Quickstart
 
- * a valid `delivery.license` file in this directory
+1. `git clone` the repo.
+1. place a valid `delivery.license` file in this directory
+1. Update .gitignore with these changes:
```

Comment:
You already added these to the `.gitignore` file, so we can eliminate them here, right?